docs/locale/fr/LC_MESSAGES/cryptography.po

```diff
@@ -163,184 +162,183 @@ msgstr ""
 "parties publiques en utilisant le serveur de métadonnées comme canal de "
 "transmission pour déduire une clé secrète partagée dans le style de DH."
 
-#: ../../cryptography.rst:36
+#: ../../cryptography.rst:39
 msgid ""
 "To prevent a malicious metadata server to modify the DH canal (Man-in-"
 "the-middle attack), Alice and Bob authenticate their secrete key using "
-"the `Short Authentication String "
-"<https://www.iacr.org/archive/crypto2005/36210303/36210303.pdf>`_ (SAS) "
-"protocol. Each party share verbally a SAS token that his pair must "
-"validate from a set of tokens `(accordingly to the recommendations of the"
-" scientific community) "
+"the `Short Authentication String <https://www.iacr.org/archive/"
+"crypto2005/36210303/36210303.pdf>`_ (SAS) protocol. Each party share "
```

Comment:
Would it be possible to not have `link`s being splited ?

Reply:
It would be possible. Wrap is done by Poedit on save (although it preserves formatting of existing files) and it can be enabled or disabled for all msgstr. 

However, if links get splitted when wrapping, I supposed it is not an issue to do so. You would prefer to keep it in one line for readability ?

Reply:
I would prefer for review, but if that require too much work then forget it.